src/platform/windows/gui/help/sysinfo.h

```diff
@@ -0,0 +1,39 @@
+#include <string>
+
+namespace gui::help {
+class SysInfo {
+    void sysinfoWindow();
+
+   public:
+    bool sysinfoWindowOpen = false;
+
+    void displayWindows();
+};
+
+enum CPUVendor {
+    VENDOR_INTEL = 0,
+    VENDOR_AMD = 1,
+};
+
+struct CPUInfo {
+    CPUVendor vendor;
+
+    // Misc
+    char cpu_string[0x21];
+    char brand_string[0x41];
+    bool OS64bit;
+    bool Mode64bit;
+
+	bool bAtom;
```

Comment:
Knowing if the app is running under a certain type of processor is useless for the emulator itself.